aporia.nim

```diff
@@ -781,6 +782,9 @@ proc addTab(name, filename: string, setCurrent: bool = true,
   ## ``-1`` is returned upon error.
   assert(win.nimLang != nil)
 
+  # Save the preferences before the tab is opened.
+  win.save()
```

Comment:
Good point. I'll move that to after the tab is opened and add a new one in the preferences.

I'm a bit unfamiliar with how Nim does this though. The current 'win' var in settings.nim won't work as that's type ptr utils.MainWin. Setting a new variable and calling save() on that, without the pointer will compile fine but doesn't function. Is there another way to do this?